scripts/apps/search/directives/ItemRepo.js

```diff
@@ -14,6 +14,16 @@ export function ItemRepo(
         link: function(scope, elem) {
 
             /*
+             * function to initialize default values on init or search provider change
+             */
+            scope.setDefaultValues = function() {
```

Comment:
This does not need to be on the scope. Generally, we only put functions that need to be "_two-way bound_" between view and model on the scope as this increases Angular digest cycle processing and reduces app performance. In this case, this is not even used in the view, so please make just a regular function here.